tests/dummy/app/templates/horizontal-bar.hbs

```diff
@@ -72,6 +75,18 @@
             <span class="help-block">Label Width Multiplier: {{labelWidthMultiplier}}</span>
           </div>
         </div>
+        <div class="form-group">
+          <div class="col-lg-offset-4 col-lg-8">
+            <div class="checkbox">
+              <label>
+                {{input type="checkbox" checked=hasAxisTitles }} Has axis titles
+              </label>
+            </div>
+          </div>
+        </div>
+        {{#if hasAxisTitles}}
```

Comment:
Remove this once you rebase with master. The `axis-title-config` template already contains this logic.